pkg/gadgets/run/tracer/run.go

```diff
@@ -527,6 +558,20 @@ func (g *GadgetDesc) getColumns(info *types.GadgetInfo) (*columns.Columns[types.
 			}
 			timestampsCounter++
 			continue
+		default:
+			enum, ok := member.Type.(*btf.Enum)
+			if !ok {
+				break
+			}
+
+			// Add raw columns for enum. Below a column that print the string
+			// representation is added as well.
+			enums = append(enums, enumDef{
+				btfEnum: enum,
+				attrs:   attrs,
+			})
+			attrs.Name = member.Name + "_raw"
+			attrs.Visible = false
```

Comment:
I think it's better to move this outside this switch case. It seems hacky to me to use the default case above for this purpose.